### ISSUE TITLE
mask = self.linear(p); mask = torch.tanh(mask)

### ISSUE THREAD

**Issue by ghost** [sweetcocoa/DeepComplexUNetPyTorch#9]
I see you put 'torch.tanh' directly on 'mask' produced by 'self.linear(p)' in the 'forward(self, bd)' of 'class UNet(nn.Module)' (unet.py, line 119).  If I read this right, you implement 'torch.tanh' on an complex number, but it implements 'torch.tanh' on the magnitude of the complex number as published in the 'Phase-Aware Speech Enhancement with Deep Complex U-Net' paper. _Are this two implementation the same?_ Or it should be  implemented like the 'mask_processing' in https://github.com/Doyosae/Phase_aware_Deep_Complex_UNet/blob/master/model_module.py (line 73)?
